client/src/app/site/motions/services/motion-pdf.service.ts

```diff
@@ -254,6 +254,29 @@ export class MotionPdfService {
             ]);
         }
 
+        // supporters
+        let minSupporters;
+        this.configService
+            .get<number>('motions_min_supporters')
```

Comment:
`.instant()`
This function was called after a button click. Subscribing here is not necessary and potentially harmful, since there is no unsub.